source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -36,11 +36,26 @@
 import requests as requests
 from jormungandr import app
 from jormungandr.exceptions import TechnicalError
-from jormungandr.utils import get_pt_object_coord
+from jormungandr.utils import get_pt_object_coord, PeriodExtremity
 from jormungandr.street_network.street_network import AbstractStreetNetworkService, StreetNetworkPathKey
 from jormungandr.ptref import FeedPublisher
 from jormungandr.fallback_modes import FallbackModes as fm
 from six import text_type
+from enum import Enum
+
+MAX_MATRIX_POINTS = 100  # The limit is fixed by Here API
+
+# Possible values to active/deactivate realtime traffic
+class RealTimeTraffic(Enum):
+    enabled = "enabled"
+    disabled = "disabled"
+
+
+# Possible values to choose a matrix method
```

Comment:
Maybe document here (or somewhere else) what these two options means (not obvious at first)